client/blocks/jitm/index.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSite } from 'state/ui/selectors';
+import Banner from 'components/banner';
+
+const unescape = ( str ) => {
+	return str.replace( /&#(\d+);/g, ( match, entity ) => String.fromCharCode( entity ) );
+};
+
+const JITM = ( props ) => {
+	if ( props.data.length === 0 || ! props.currentSite ) {
+		return null;
+	}
+
+	const jitm = props.data[ 0 ];
```

Comment:
Should we add an `Array.isArray()` here or earlier before trying to get an element from the array?

Reply:
Good call